frappe/__init__.py

```diff
@@ -2111,7 +2126,21 @@ def get_print(
 	finally:
 		local.form_dict = original_form_dict
 
-	return get_pdf(html, options=pdf_options, output=output) if as_pdf else html
+	if not as_pdf:
+		return html
+
+	if local.form_dict.chrome_pdf_generator:
+		hook_func = frappe.get_hooks("chrome_pdf_generator")
```

Comment:
```suggestion
		hook_func = frappe.get_hooks("pdf_generator")
```
Let's call it just `pdf_generator`? Default pdf_generator is "WKHTML2PDF", your will be "CHROME"